### DISCUSSION TITLE
(Help me) Please tell me about Japanese summer festivals you would like to visit !!!

### DISCUSSION THREAD

**Discussion by Idescrew** [NAIST-SE/AttractingContributors#1178] [2024 Attraction Contributions]
https://github.com/Idescrew/AttractingContributors

Please tell me about Japanese summer festivals you would like to visit !!!

**Comment by honamikoyama** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
https://github.com/honamikoyama/Travel.git
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1182

**Reply by Idescrew** [reply]
Thank you
I write place I want to visit
